app/AndroidManifest.xml

```diff
@@ -527,8 +552,9 @@
         <activity android:name="org.commcare.activities.PromptCczUpdateActivity"/>
         <activity
             android:label="@string/title_data_change_logs_activity"
-            android:exported="false"
-            android:name="org.commcare.activities.DataChangeLogsActivity">
+            android:exported="true"
```

Comment:
why do we need to export this activity ? (I don't recall using it in callouts)

Reply:
It's not, but it's supposed to respond to `ACTION_VIEW` actions, so I think it needs to be exported. I didn't test this though, let me do that.

Reply:
Btw from the [Spec](https://docs.google.com/document/d/1LJIaQMoex6SM90gAxJbLRsMw8FC_0FtJLbMbmxGE5dE/edit?tab=t.0), I don't see the reason for this activity to respond o `ACTION_VIEW` actions, can you provide some context here?

Reply:
I think that's again because of dev tooling - https://github.com/dimagi/commcare-android/blob/master/scripts/ccc#L217 which I am fine with killing.

Reply:
@shubham1g5 I've reverted the `exported` status of these components, I will kill them or move to a different manifest file in a different PR.